lib/src/services/controllers/match_controller.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:rescado/src/data/models/like.dart';
+import 'package:rescado/src/data/repositories/group_repository.dart';
+import 'package:rescado/src/utils/logger.dart';
+
+final matchesControllerProvider = StateNotifierProvider<MatchesController, AsyncValue<List<Like>>>(
+  (ref) => MatchesController(ref.read).._initialize(),
+);
+
+class MatchesController extends StateNotifier<AsyncValue<List<Like>>> {
```

Comment:
Another single file name vs plural class name and by consequence incorrect name passed as argument to the logger below.